packages/core/admin/admin/src/components/GuidedTour/utils/setCurrentStepToLocaleStorage.js

```diff
@@ -0,0 +1,5 @@
+const setCurrentStepToLocaleStorage = step => {
```

Comment:
Can you create a single file where you'll put all the logic regarding the localStorage of the guided tour?


Something like `persistStateToLocalStorage`

```js
export const CURRENT_STEP = 'GUIDED_TOUR_CURRENT_STEP';  // reuse the constant in your `init.js` file.
export const COMPLETED_STEPS = 'GUIDED_TOUR_COMPLETED_STEPS';
const parse = JSON.parse;
const stringify = JSON.stringify;

const persistStateToLocalStorage = {
  clear() {
    localStorage.removeItem(CURRENT_STEP);
    localStorage.removeItem(COMPLETED_STEPS); 
  },

  addCompletedStep: completedStep => {
    const currentSteps =  parse(localStorage.getItem(COMPLETED_STEPS)).slice() // Use the shallow copy
    
    // ...

    currentSteps.push(completedStep);

    localStorage.setItem(COMPLETED_STEPS, stringify(currentSteps);
  },

  // add the other methods
};
```

Reply:
Also you'll need to make so update to this [file](https://github.com/strapi/strapi/blob/master/packages/core/helper-plugin/lib/src/utils/auth/index.js) in order to not remove the keys when logging out